app/src/main/java/jp/developer/retia/frozenword/ui/activity/AddHabbitActivity.kt

```diff
@@ -34,34 +43,46 @@ class AddHabbitActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        lifecycleScope.launch {
-            repeatOnLifecycle(Lifecycle.State.STARTED) {
-                mainViewModel.uiState.collect { uiState ->
-                    when (uiState) {
-                        is AddHabbitUiState.HabbitTitlePage -> show1stPane(
-                            uiState.title,
-                            uiState.sampleTitle
-                        )
-                        is AddHabbitUiState.ShortHabbitTitlePage -> TODO()
-                        is AddHabbitUiState.TitlePage -> TODO()
-                    }
-                }
-            }
+//        lifecycleScope.launch {
+//            repeatOnLifecycle(Lifecycle.State.STARTED) {
+//                mainViewModel.uiState.collect { uiState ->
+//                    when (uiState) {
+//                        is AddHabbitUiState.HabbitTitlePage -> show1stPane(
+//                            uiState.title,
+//                            uiState.sampleTitle
+//                        )
+//                        is AddHabbitUiState.ShortHabbitTitlePage -> TODO()
+//                        is AddHabbitUiState.TitlePage -> TODO()
+//                    }
+//                }
+//            }
+//        }
+        setContent {
+            AddHabbitScreen()
         }
+//        lifecycleScope.launch {
+//            repeatOnLifecycle(Lifecycle.State.STARTED) {
+//                mainViewModel.events.collect { event ->
+//                    when (event) {
+//                        is AddHabbitEvent.Back -> {
+////                            onBackPressed()
```

Comment:
```suggestion
// //                            onBackPressed()
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>